python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/aaa-server-groups-tacacs-plus.j2

```diff
@@ -4,10 +4,10 @@
  that can be found in the LICENSE file.
 #}
 {# eos - aaa server groups #}
-{% for aaa_server_group in aaa_server_groups | arista.avd.natural_sort('name') | arista.avd.natural_sort('type') %}
-{%     if aaa_server_group.type == 'tacacs+' %}
+{% for aaa_server_group in aaa_server_groups | arista.avd.natural_sort('name') %}
```

Comment:
I think you can save some if statement using
```suggestion
{% for aaa_server_group in aaa_server_groups | selectattr('type', 'equalto', 'tacacs+') | arista.avd.natural_sort('name') %}
```

Reply:
After implementing this, it is failing with reason `aaa_server_groups` is undefined. It required an outer if-endif to check `aaa_server_groups` is defined or not.

Reply:
or you can `sort` and then `selectattr` though it is sad to have to sort things we don't need but thats what we are already doing

Reply:
This worked. Done.